src/cmake/stir_exe_targets.cmake

```diff
@@ -22,6 +22,10 @@ foreach(source ${${dir_EXE_SOURCES}})
   if(BUILD_EXECUTABLES)
    get_filename_component(executable ${source} NAME_WE)
    add_executable(${executable} ${ALL_HEADERS} ${ALL_INLINES}  ${ALL_TXXS} ${source} $<TARGET_OBJECTS:stir_registries>)
+if (STIR_WITH_TORCH)
+   target_link_libraries(${executable}  ${TORCH_LIBRARIES})
+   target_include_directories(${executable} PUBLIC ${TORCH_INCLUDE_DIRS})
+endif()
```

Comment:
irrelevant comment for now, but do we really need this for all exes? The mod to src/buildblock/CMakeLists.txt should be enough (due to transitive dependence)

Reply:
Probably not ... but let's see :)